config/index.js

```diff
@@ -417,7 +417,7 @@ var conf = convict({
   signinConfirmation: {
     enabled: {
       doc: 'enable signin confirmation',
-      default: false,
+      default: true,
```

Comment:
Just noting that we need to be careful about merging this on-by-default config to master...